invenio_accounts/models.py

```diff
@@ -87,3 +89,17 @@ def validate_ip(self, key, value):
         if value == 'untrackable':  # pragma: no cover
             value = None
         return value
+
+
+class SessionActivity(db.Model, Timestamp):
+    """User Session Activity model."""
+
+    __tablename__ = "accounts_user_session_activity"
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    user_id = db.Column(db.Integer, db.ForeignKey(User.id))
+
+    sid_s = db.Column(db.String(255), unique=True, index=True)
```

Comment:
Both -- unserializing the `sid_s` produces the session.
Sure, I can add some docstrings. I didn't know any of this when I created the model. It's good that you're pointing out stuff that's not immediately apparent, as I'd have a hard time doing the same.